ruler-models/src/commonMain/kotlin/com/spotify/ruler/models/ResourceType.kt

```diff
@@ -0,0 +1,11 @@
+package com.spotify.ruler.models
```

Comment:
This file is missing the license header.

Reply:
We don't lint against that right now, but I think that it might be possible. Maybe worth looking into this in the future.